ext/js/accessibility/google-docs-xray.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023  Yomitan Authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/** Entry point. */
+function main() {
+    /** @type {Window} */
+    // @ts-expect-error - Firefox Xray vision
+    const window2 = window.wrappedJSObject;
+    if (!(typeof window2 === 'object' && window2 !== null)) { return; }
+    // The extension ID below is on an allow-list that is used on the Google Docs webpage.
+    // @ts-expect-error - Adding a property to the global object
```

Comment:
Would it be better to extend the window's type definition using declare global? At least that way we can get type checking on what we are trying to store in the variable, and avoid issues like variable typos or logic errors doing a refactor.

Reply:
It could be but I didn't because it was already a one-off (now a two-off) thing in the code, that realistically has no need to be exposed everywhere else in the codebase, which will happen if a typescript interface is added.